.circleci/install-lint.sh

```diff
@@ -10,6 +10,6 @@ function error() {
 
 trap error SIGINT
 
-curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.35.2
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.45.2
```

Comment:
another option (not necessarily in this PR) would be to use the `golangci-lint` docker image directly, which I imagine would work here...